### PR TITLE
Fix English action links

### PR DESCRIPTION
**Pull request by roj1512**
- They were broken by merging #355 in an accident.

### CODE REVIEW

**Review by KnorpelSenf — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
I'll wait for the build to complete before we merge

**Comment by roj1512** [comment]
> the build to complete

Fair enough.

**Pull request merged**
